admin-dev/themes/default/scss/partials/_switch-btn.scss

```
@@ -103,7 +103,7 @@
       }
 
       ~ .slide-button {
-        background: $brand-success;
+        background: $primary-color;
```

```suggestion
        background: $brand-success;
```

Can you commit this change ? 
Like @NeOMakinG said "it looks like we don't want to change switches"

Your PR will be approved to fix the issue about preview image theme